parlai/mturk/core/dev/worker_manager.py

```diff
@@ -334,20 +335,10 @@ def _get_agent_from_pkt(self, pkt):
             self._log_missing_agent(worker_id, assignment_id)
         return agent
 
-    def change_agent_conversation(self, agent, conversation_id, new_agent_id):
-        """Handle changing a conversation for an agent, takes a callback for
-        when the command is acknowledged
+    def register_to_conv(self, agent, conversation_id):
```

Comment:
This is the only functionality that `worker_manager` actually needs to be capable or aware of in what used to take place in `change_agent_conversation`.